docs/creating_integration_manifest.md

```diff
@@ -25,6 +25,7 @@ Or a minimal example that you can copy into your project:
   "domain": "your_domain_name",
   "name": "Your Integration",
   "documentation": "https://www.example.com",
+  "issue_tracker": "https://github.com/example/my_project/issues",
```

Comment:
let's not add it to the minimal example but the full example.

Reply:
Adjusted. 👍